spinalcordtoolbox/utils/validation.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# Validation related utilites
+
+
+def check_dimensions_match(input_im, im_vertebral_labeling):
```

Comment:
@joshuacwnewton @kousu what do you think about this new utils/validation file?

Reply:
Oh uh I have no idea. let me look.

Reply:
I think you could do without this since so far its contents can be simplified to a one-liner. Are there other kinds of validation we have in mind?

Reply:
> I think you could do without this since so far its contents can be simplified to a one-liner. 

I agree with that.

> Are there other kinds of validation we have in mind?

Currently, I don't have other validations in mind, but if we have some in the future, we can add more validations functions here, which could be easier for the management of all validations. Please see the examples from https://github.com/neuropoly/spinalcordtoolbox/pull/3120#discussion_r549174620